### PR TITLE
[bugfix] preserve state of skipLoggingPayload in the client Authentic…

### PR DESCRIPTION
**Pull request by akinross**
…ate method

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -429,6 +430,14 @@ func (c *Client) MakeRestRequest(method string, rpath string, body *container.Co
 
 // Authenticate is used to
 func (c *Client) Authenticate() error {
+	// Setting skipLoggingPayloadState to preserve state during call of the method
+	skipLoggingPayloadState := c.skipLoggingPayload
+
+	// Setting skipLoggingPayload to always show in logs that Authenticate method is called
+	c.skipLoggingPayload = false
```
Comment (lhercot) [inline client/client.go:437]:
```suggestion

```

**Review by lhercot — CHANGES_REQUESTED** [review]

File: client/client.go [hunk 2]
```diff
@@ -316,9 +316,10 @@ func (c *Client) useInsecureHTTPClient(insecure bool) *http.Transport {
 }
 
 // Takes raw payload and does the http request
-//  Used for login request
-//  passwords with special chars have issues when using container
-//  for encoding/decoding
+//
+//	Used for login request
+//	passwords with special chars have issues when using container
+//	for encoding/decoding
```
[the remark this answers is not in the document]
Comment (akinross) [reply client/client.go:322]: yes

Comment (anvitha-jain) [inline client/client.go:322]: If these are two different sentences, can it start with capital letter and with a full stop?

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

**Comment by shrsr** [comment]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**